src/import-data/ImportData.test.ts

```diff
@@ -302,50 +300,46 @@ describe('ImportData', () => {
       // Arrange
       const user = userEvent.setup();
 
+      asMockedFn(fetchDataCatalog).mockResolvedValue([
+        {
+          id: 'aaaabbbb-cccc-dddd-eeee-ffffgggghhhh',
+          'dct:creator': 'testowner',
+          'dct:description': 'A test snapshot',
+          'dct:identifier': '00001111-2222-3333-aaaa-bbbbccccdddd',
+          'dct:issued': '2023-10-02T11:30:00.000000Z',
+          'dct:title': 'test-snapshot-1',
+          'dcat:accessURL':
+            'https://jade.datarepo-dev.broadinstitute.org/snapshots/details/00001111-2222-3333-aaaa-bbbbccccdddd',
+          'TerraDCAT_ap:hasDataCollection': [],
+          accessLevel: 'reader',
+          storage: [],
+          counts: {},
+          samples: {},
+          contributors: [],
+        },
+        {
+          id: '11112222-3333-4444-5555-666677778888',
+          'dct:creator': 'testowner',
+          'dct:description': 'Another test snapshot',
+          'dct:identifier': 'aaaabbbb-cccc-1111-2222-333333333333',
+          'dct:issued': '2023-10-02T11:30:00.000000Z',
+          'dct:title': 'test-snapshot-2',
+          'dcat:accessURL':
+            'https://jade.datarepo-dev.broadinstitute.org/snapshots/details/aaaabbbb-cccc-1111-2222-333333333333',
+          'TerraDCAT_ap:hasDataCollection': [],
+          accessLevel: 'reader',
+          storage: [],
+          counts: {},
+          samples: {},
+          contributors: [],
+        },
+      ]);
+
       const queryParams = {
         format: 'snapshot',
         snapshotIds: ['00001111-2222-3333-aaaa-bbbbccccdddd', 'aaaabbbb-cccc-1111-2222-333333333333'],
       };
-      const { getWorkspaceApi, importSnapshot } = setup({ queryParams });
-
-      asMockedFn(useDataCatalog).mockReturnValue({
```

Comment:
Had to move this mock before `setup`. Kind of surprised that it worked before...